scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -151,43 +145,27 @@ export class SpreadsheetStandardizer {
       }
     }
 
-    return output;
+    // Enforce a specific property order on the output for better debugging.
+    return Object.fromEntries(
+      FIELD_ORDER.map((key: string) => [
+        key,
+        key in output ? output[key as keyof typeof output] : undefined,
+      ]).filter(([, val]) => val !== undefined),
+    );
   }
 
-  retrieveLowIncomeProgram(authority: string, state: string) {
+  retrieveLowIncomeProgram(
+    authority: string,
+    state: string,
+  ): LowIncomeAuthority | undefined | null {
     if (this.lowIncomeThresholds![state] === undefined) {
       return undefined;
     }
-    return authority in this.lowIncomeThresholds![state];
-  }
-
-  createAmount(input: Record<string, string>): Record<string, number | string> {
-    const amount: Record<string, number | string> = {
-      type: input['amount.type'],
-      number: +cleanDollars(input['amount.number']),
-    };
-    if (
-      input['amount.representative'] !== undefined &&
-      input['amount.representative'] !== ''
-    ) {
-      amount.representative = +cleanDollars(input['amount.representative']);
-    }
-    if (
-      input['amount.minimum'] !== undefined &&
-      input['amount.minimum'] !== ''
-    ) {
-      amount.minimum = +cleanDollars(input['amount.minimum']);
-    }
-    if (
-      input['amount.maximum'] !== undefined &&
-      input['amount.maximum'] !== ''
-    ) {
-      amount.maximum = +cleanDollars(input['amount.maximum']);
+    if (authority in this.lowIncomeThresholds![state]) {
```

Comment:
question: what does the syntax here mean: this.lowIncomeThresholds![state] vs this.lowIncomeThresholds[state]?